.pre-commit-config.yaml

```diff
@@ -2,9 +2,6 @@ ci:
   autofix_prs: false
   autoupdate_schedule: monthly
 
-default_language_version:
-  node: 16.15.0
-
```

Comment:
If this PR runs into any problems, it might be because we removed these lines.  We needed to add them here for some reason, back in the day, but we shouldn't need them anymore.